### ISSUE TITLE
The website doesn't open

### ISSUE THREAD

**Issue by ghost** [scaron/prettyphoto#172]
Seems like it's having trouble connecting to database.

**Comment by helgatheviking** [comment]
This is still true in 2020, which is not a good sign.

**Comment by gardavis** [comment]
Yes - the link is https://www.no-margin-for-errors.com/ and the error is: 

Error establishing a database connection
